xarray/tests/test_plot.py

```diff
@@ -131,8 +134,9 @@ def setup(self):
         # Remove all matplotlib figures
         plt.close("all")
 
-    def pass_in_axis(self, plotmethod):
-        fig, axes = plt.subplots(ncols=2)
+    def pass_in_axis(self, plotmethod, subplot_kw=None):
+        subplot_kw = {} if subplot_kw is None else subplot_kw
```

Comment:
```suggestion
```
I just realised that the default is `plt.subplots(..., subplot_kw=None)` - so this is actually not necessary.